src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,34 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder str = new StringBuilder();
```

Comment:
Name your variables more informative
```suggestion
        StringBuilder report = new StringBuilder();
```